src/utils/SortMembers.ts

```diff
@@ -96,13 +97,14 @@ interface IMemberScore {
     numRooms: number;
 }
 
-export function buildMemberScores(cli: MatrixClient): { [key: string]: IMemberScore | undefined } {
+export function buildMemberScores(cli: MatrixClient): { [key: string]: IMemberScore } {
```

Comment:
Minor point - not sure if we prefer `<Record<string, ...>` for cases like this (also valid elsewhere in this file)

Reply:
In a case where the key string is uselessly `key` I agree, but IMO this form is better as the key can be given a helpful name, I'll update it to `userId` for self-documenting

Reply:
I'd never considered that, makes sense